### ISSUE TITLE
Allegro delivery - missing tracking info

### ISSUE THREAD

**Issue by DHSC1** [allegro/allegro-api#10438]
Hello,
We have requested (command id ef730ae4-4327-409d-b43b-4f2390ba6b85) a label via the api for an order (b9e3a550-afda-11ef-8a51-51353ee2a494) and downloaded it to our system where we have it stored including tracking information (Z1019920320). We found that for some reason the tracking information disappeared (not that it was changed, but that it disappeared) from the allegro portal. Can you please investigate this? Allegro downgraded our rating because of that. 
We tried to address this with support first, but they referred us here.
Thank you.

![image](https://github.com/user-attachments/assets/2a900cb5-ac59-455b-abf9-7bcdf29d6ef6)

**Comment by MaciejFrackowiak** [comment]
I need to consult the team on your case. I will be back with an answer.

**Comment by MaciejFrackowiak** [comment]
Yes, we didn't send information about the tracking number to the selected order.
